app/page.tsx

```diff
@@ -1,14 +1,42 @@
-"use client" // Add this line at the top
+'use client';
 
-import CalculTVA from "../components/business/calculate-tva"
+import React, { useState } from 'react';
+import Navigation from "@/components/Navigation";
+import CalculTVA from "../components/CalculTva";
```

Comment:
why did you used relative module import instead of module import with an alias ?

Reply:
I just had some internal issue in my local, ll refactor them all to use an alias.